releasecalendar/client_test.go

```diff
@@ -142,7 +154,7 @@ func TestGetLegacyRelease(t *testing.T) {
 		})
 	})
 
-	Convey("Given that 200 OK is returned by the API with an invalid body", t, func() {
+	Convey("Given that 500 StatusInternalServerError is returned by the API with an invalid body", t, func() {
```

Comment:
Is this change correct? It looks like the test client does return 200 OK?

Reply:
yep sorry I got confused as to who was returning the error